src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -316,6 +324,16 @@ object Writer {
                       runtimeDeps = runtime_deps -- uvexports,
                       processorClasses = getProcessorClasses(u),
                       licenses = licenses)
+                  case Language.Kotlin =>
```

Comment:
same as Java, let's share,

matching: `case Language.Java | Language.Kotlin =>` should just work.